Sources/Sentry/SentrySpan.m

```diff
@@ -101,6 +124,12 @@ - (NSDictionary *)serialize
         }
     }
 
+    if (_tags != nil) {
+        @synchronized(_tags) {
+            [mutableDictionary setValue:_tags.copy forKey:@"tags"];
+        }
+    }
```

Comment:
`m`: I think the `_tags` will never be `nil`. Instead, we should check if they are empty.
```suggestion
    @synchronized(_tags) {
        if (_tags.count > 0) {
            [mutableDictionary setValue:_tags.copy forKey:@"tags"];
        }
    }
```

Reply:
Why did we resolve that?

Reply:
Because I followed your suggestion. Don't know why GitHub is not showing the outdated tag here.

Reply:
![image](https://user-images.githubusercontent.com/1633368/127241649-fea8c11b-4e60-4a57-ba43-7f309e45425a.png)